### PR TITLE
update csv to beta channel

### PR DESCRIPTION
**Pull request by hbradfield**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/38050

### CODE REVIEW

File: deploy/olm-catalog/ibm-auditlogging-operator/ibm-auditlogging-operator.package.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 channels:
 - currentCSV: ibm-auditlogging-operator.v3.6.0
   name: dev
+- currentCSV: ibm-auditlogging-operator.v3.6.0
+  name: beta
 - currentCSV: ibm-auditlogging-operator.v3.5.0
   name: stable-v1
 defaultChannel: stable-v1
```
Comment (znhafeez) [inline deploy/olm-catalog/ibm-auditlogging-operator/ibm-auditlogging-operator.package.yaml:8]: default channel here should be `beta`

File: deploy/olm-catalog/ibm-auditlogging-operator/3.6.0/ibm-auditlogging-operator.v3.6.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -49,7 +50,7 @@ metadata:
     capabilities: Basic Install
     categories: Security
     certified: "false"
-    containerImage: quay.io/opencloudio/ibm-auditlogging-operator:latest
+    containerImage: quay.io/opencloudio/ibm-auditlogging-operator:3.6.0
```
Comment (znhafeez) [inline deploy/olm-catalog/ibm-auditlogging-operator/3.6.0/ibm-auditlogging-operator.v3.6.0.clusterserviceversion.yaml:53]: should you be using the sha here instead?

Comment (hbradfield) [reply deploy/olm-catalog/ibm-auditlogging-operator/3.6.0/ibm-auditlogging-operator.v3.6.0.clusterserviceversion.yaml:53]: this is in the annotations for the ocp console ui to display. was following swati's pr

**Comment by znhafeez** [comment]
lgtm

**Pull request merged**